guides/release/components-update/templates-are-html.md

```diff
@@ -0,0 +1,198 @@
+      </li>
+    </ul>
+
+    <p>Let me know when you've nailed down the dates!</p>
+  </section>
+
+  <form>
+    <input>
+    <button>
+      Send
+    </button>
+  </form>
+</div>
+```
+
+And if you serve the app, you'll see the HTML rendered, though it will be
+unstyled. You can copy these styles as well to `app/styles/app.css` to style the
+example components:
+
+```css {data-filename=styles/app.css}
+body {
+  max-width: 800px;
+  margin: auto;
+  padding: 2em;
+  font-family: sans-serif;
+  background-color: #fdfdfd;
+}
+
+.messages {
+  display: grid;
+  grid-template-columns: 80px 1fr;
+  padding: 2em;
+  border-radius: 0.5em;
+  box-shadow: 0 0.25em 1.5em 0.25em rgba(0,0,0,0.1);
+}
+
+.messages > section {
+  margin-bottom: 1.5em;
+  line-height: 1.5em;
+}
+
+.messages p,
+.messages ul,
+.username {
+  margin: 0.5em 0;
+}
+
+.local-time {
+  font-size: 0.8em;
+  color: #da6c4d;
+  font-weight: normal;
+  margin-left: 10px;
+}
+
+.avatar {
+  position: relative;
+  border-radius: 50%;
+  width: 60px;
+  height: 60px;
+
+  text-align: center;
+  line-height: 60px;
+
+  color: white;
+  font-weight: bold;
+  background-color: #ff907b;
+}
+
+.avatar.is-active:after {
+  content: " ";
+  height: 14px;
+  width: 14px;
+  border: solid 3px white;
+  border-radius: 50%;
+  background-color: #8bc34a;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+
+.current-user .avatar {
+  background-color: #30aba5;
+}
+
+form {
+  display: grid;
+  grid-column: span 2;
+  grid-template-columns: 1fr 6em;
+}
+
+form > input {
+  padding: 0.5em;
+  border-top-left-radius: 0.5em;
+  border-bottom-left-radius: 0.5em;
+  border: 1px solid #cccccc;
+  border-right: none;
+  font-size: 1em;
+}
+
+form > button {
+  border-top-right-radius: 0.5em;
+  border-bottom-right-radius: 0.5em;
+  border: 1px solid #cccccc;
+  font-size: 1em;
+}
+```
+
+![screenshot of styled message app](/images/ember-core-concepts/messaging-app-1.png)
+
+You start building parts of an Ember application using HTML, so if you already know HTML and CSS, you know how to build a basic Ember application!
+
+You can even use SVG or web components without any changes. As long as your HTML is valid, Ember will render it.
+
+# Self-Closing Tags
+
+In addition to normal HTML syntax, Ember allows you to use self-closing syntax (`<div />`) as a shorthand for an opening and closing tag (`<div></div>`).
+
+You don't **need** to use this syntax for "void" HTML tags such as `img` or `br`, which are already defined as self-closing by the HTML spec, but you **can** use this syntax as a shorthand for tags that are not self-closing.
+
+# Supported Features
+
+This means that all of the following HTML features work as-is:
+
+- Web components
+- SVG
+- HTML comments
+- Whitespace has the same rules as normal HTML
+- Special HTML elements like `<table>` and `<select>`
+
+# Restrictions
+
+There are a handful of restrictions on the HTML that you can put in an Ember template:
+
+- Valid HTML only
```

Comment:
This is a little confusing to me, because you're not actually writing the `<body>` tag. It's more like:

```
Only HTML elements that are valid in a `<body>` tag can be used, and the content must be valid HTML
```

Reply:
I like your re-written version. Let's go with that 👍